src/main/java/com/example/batteryprice/service/BatteriesPriceService.java

```diff
@@ -53,11 +52,11 @@ public Battery getValueAndCalculatePrice(BatteriesInRangeDto batteriesDto) throw
 
     }
 
-    public KWPriceDto getValueRestCall() {
+    public KWPriceDto getValueRestCall(PriceOperation priceOperation) {
         return webClient
                 .get().
-                uri("/priceKw").
-                retrieve()
+                uri("/"+priceOperation.toString())
```

Comment:
I try to use query string in mock server but without success, so i create 3 different mock serv with path localhost:port/operation